docker/build_and_run.sh

```diff
@@ -4,5 +4,5 @@ docker build --build-arg https_proxy=$https_proxy \
 
 # Another container should expose AESM and its socket in aesmd-socket volume.
 # Replace /dev/sgx/enclave with /dev/isgx if you use the Legacy Launch Control driver
-docker run --env http_proxy --env https_proxyi --device=/dev/sgx/enclave -v aesmd-socket:/var/run/aesmd -it ehsmcore bash
+docker run --env http_proxy --env https_proxyi --device=/dev/sgx/enclave -v aesmd-socket:/var/run/aesmd -it -P ehsmcore
```

Comment:
--env https_proxyi is a typo.

correct it as "--env https_proxy"

Reply:
what's the "-P ehsmcore" means?

Reply:
-P 
Container internal ports are randomly mapped to host ports